### DISCUSSION TITLE
Very confused, how do I convert a folder of images to another format?

### DISCUSSION THREAD

**Discussion by Maple38** [ImageMagick/ImageMagick#5676] [Help] [answered]
I really tried doing this on my own, but I couldn't figure it out. All I'm looking to do is convert a folder full of .webp images to .jpg. Could somebody explain how?

**Comment by fmw42** [comment, marked as answer]
What version of Imagemagick?  Do you have support for WEBP?

Assume IM 6

Changed directories to the one with the images and make a backup copy, since mogrify in this form will write over the files

`mogrify -format jpg *.webp`

If on IM 7, add magick before mogrify

if you have an empty folder and want the output there, then

`mogrify -path path_to/new_folder -format jpg *.webp`

**Comment by Maple38** [comment]
Thanks so much, it worked.
